hibernate-core/src/main/antlr/org/hibernate/grammars/hql/HqlParser.g4

```diff
@@ -1663,7 +1657,7 @@ jsonValueReturningClause
 	;
 
 jsonValueOnErrorOrEmptyClause
-	: (ERROR | NULL | DEFAULT expression) ON (ERROR | EMPTY)
+	: ( ERROR | NULL | ( DEFAULT expression ) ) ON (ERROR|EMPTY)
```

Comment:
Please roll back this change;it's inconsistent with the formatting of the rest of the BNF.

Reply:
And the extra parens you added are unnecessary.